ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productcarousel/v1/productcarousel/productcarousel.html

```diff
@@ -0,0 +1,28 @@
+<!--
+
+    Copyright 2019 Adobe. All rights reserved.
+    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License. You may obtain a copy
+    of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under
+    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+    OF ANY KIND, either express or implied. See the License for the specific language
+    governing permissions and limitations under the License.
+
+-->
+<sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html" data-sly-call="${clientlib.all @ categories='aem.venia.productcarousel'}" />
+<sly data-sly-test="${!properties.product}">
+    <div class="placeholder__empty product__carousel__title" data-sly-text="${component.title}"></div>
+</sly>
+<sly data-sly-use.productTpl="product.html" data-sly-test="${properties.product}">
+    <div data-comp-is="productcarousel" class="product__carousel carousel__root">
+        <div class="product__carousel__parent" data-sly-use.productList="${'com.adobe.cq.commerce.core.components.models.productcarousel.ProductCarousel' @productSkuList=properties.product}">
```

Comment:
According to BEM notation this class should be `productcarousel__parent` - http://getbem.com/naming/. The format is `block__element-modifier`.

For example the `cards__container` should be `productcarousel__cardscontainer`